libs/shared/src/containers/AccountMap/reducer.ts

```diff
@@ -24,41 +25,31 @@ export class AccountMap extends BaseReducer<IAccountStoreAPI, AccountMapState>('
       if (this.state.map[address] === undefined)
         throw new Error(`Cannot set activeAccount: ${address} does not exist`);
     }
-
+    log.trace({address}, 'Set active to {address}');
     this.draftState.active = address
   }
 
-  // Add a new account, optionally store in LocalStorate, in unlocked state
-  addAccount(name: string, address: string, signer: Signer, store: boolean = true, setActive: boolean = true, unlocked?: Wallet) {
-    // Check the signer & unlocked are actually the same account
-    if (unlocked && NormalizeAddress(address) != NormalizeAddress(unlocked?.address)) {
-      throw new Error("Accounts being stored are mis-matched");
-    }
+  // Add a new account
+  addAccount(name: string, address: string, signer: Signer) {
     // Create the account with default values
     const newAccount = buildNewAccount(name, address, signer);
-    // If asked to store, push to storage
-    if (store) {
-      storeAccount(newAccount);
-    }
-
-    // If we have an unlocked account, replace the signer
-    // (after the account has been pushed to storage)
-    if (unlocked) {
-      newAccount.signer = unlocked;
-    }
+    log.trace({address: newAccount.address}, 'Adding account: {address}');
+    // push to storage
+    storeAccount(newAccount);
```

Comment:
All pathways currently store account.  We should revisit this in the future, for example using on shared computers should not persist info